vault.yml

```diff
@@ -0,0 +1,16 @@
+$ANSIBLE_VAULT;1.1;AES256
```

Comment:
It will be good to create an example env directory and put how a default unencrypted vault-file, inventory-file and any other env specific files.

Reply:
Good idea. I'm rather going to create a new branch to do this refactor. I'll then update the Readme also accordingly